docs/src/tutorial-kinds-of-failure.md

```diff
@@ -45,33 +45,20 @@ But we're able to check this unsafe code with Kani:
 ```
 # kani src/bounds_check.rs --harness bound_check
 [...]
-** 15 of 448 failed
+SUMMARY:
+ ** 1 of 459 failed
```

Comment:
Maybe remove this and add the check that fails instead. This number is likely to change and hard to keep up-to-date.

Reply:
Removed and adjusted the text a little bit, please check how it looks.